File: app/src/main/java/fergaral/datetophoto/utils/SameWidthHeightFrameLayout.java
package fergaral.datetophoto.utils;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.FrameLayout;

/**
 * Created by fer on 16/08/15.
 */
public class SameWidthHeightFrameLayout extends FrameLayout {
    public SameWidthHeightFrameLayout(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec);
    }
}

File: app/src/main/java/fergaral/datetophoto/utils/SameWidthHeightFrameLayout.kt
package fergaral.datetophoto.utils

import android.content.Context
import android.util.AttributeSet
import android.widget.FrameLayout

/**
 * Created by fer on 16/08/15.
 */
class SameWidthHeightFrameLayout(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs) {

    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec)
    }
}